### PR TITLE
Completed main tests

### PR DESCRIPTION
**Pull request by Rafsan7238**
Reached 100% main test coverage.

### CODE REVIEW

File: tests/test_main.py [hunk 1]
```diff
@@ -95,7 +99,10 @@ def test_add_fail():
 
 
 def copy_NC_036113(url, path):
```
Comment (rbturnbull) [inline tests/test_main.py]: This function should have a different name now.

**Review by rbturnbull — CHANGES_REQUESTED** [review]
Can you please change the name of the function`copy_NC_036113` and make it more general? maybe just 'copy_dummy_fasta'

File: tests/test_main.py (hunk 1 above)
Comment (Rafsan7238) [reply tests/test_main.py]: It has been done now.

**Comment by Rafsan7238** [comment]
Incorporated the requested function name change. Since the test cases were approved before, merging the branch.

**Pull request merged**